learn.md

```diff
@@ -11,25 +11,75 @@ There are a handful of websites where you can interactively run Scala code in yo
-   specialization has the following courses.
-    * [Functional Programming Principles in Scala](https://www.coursera.org/learn/progfun1),
-    * [Functional Program Design in Scala](https://www.coursera.org/learn/progfun2),
-    * [Parallel programming](https://www.coursera.org/learn/parprog1),
-    * [Big Data Analysis with Scala and Spark](https://www.coursera.org/learn/scala-spark-big-data),
-    * [Functional Programming in Scala Capstone](https://www.coursera.org/learn/scala-capstone),
- * [Programming Reactive Systems](https://www.coursera.org/learn/scala-reactive) (also available on [edX](https://www.edx.org/course/programming-reactive-systems)).
-
-Note : On Coursera and edX, there is a paid version available. The only difference with the free version is that
-the paid version delivers a certificate of completion. Learn more about
+The following online courses provide two main paths to learn Scala. The
+fast path consists of taking the course [Effective Programming in Scala],
+otherwise you can take the full [Scala Specialization], which is made of
+four courses and a capstone project.
+
+All the courses are available for free. Optionally, a subscription gives
+you access to a certificate of completion that attests your accomplishments.
+Learn more about
 [Coursera certificates](https://learner.coursera.help/hc/en-us/articles/209819053-Get-a-Course-Certificate) or
 [edX certificates](https://support.edx.org/hc/en-us/categories/115002269627-Certificates).
 
+### Effective Programming in Scala
+
+[Effective Programming in Scala] teaches non-Scala programmers everything
+they need to be ready to work in Scala. At the end of this hands-on course,
+you will know how to achieve common programming tasks in Scala (e.g.,
+modeling business domains, implementing business logic, designing large
+systems made of components, handling errors, manipulating data, running 
+concurrent tasks in parallel, testing your code). You can learn more about
+this course in the following video:
+
+<div style="text-align: center">
+  <iframe width="560" height="315" src="https://www.youtube.com/embed/MSDJ7ehjrqo" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</div>
+
+After taking this course, you might be interested in growing further your
+skills in specific areas by taking the courses [Parallel Programming],
+[Big Data Analysis with Scala and Spark], or [Programming Reactive Systems].
+
+### Scala Specialization
+
+The [Scala Specialization] provides a hands-on introduction to functional programming using Scala. You can access the courses
+material and exercises by either signing up for the specialization or auditing the courses individually. The
+   specialization has the following courses.
+* [Functional Programming Principles in Scala],
```

Comment:
I never used whitespace in markdown url shortcuts. Did you check that this links correctly?

Reply:
Yes :)